snow/consensus/snowball/binary_snowflake.go

```diff
@@ -78,7 +95,7 @@ func (sf *binarySnowflake) Finalized() bool {
 
 func (sf *binarySnowflake) String() string {
 	return fmt.Sprintf("SF(Confidence = %d, Finalized = %v, %s)",
-		sf.confidence,
+		sf.confidence[0],
```

Comment:
This will be in the follow up PR. I set it to include the 0th element, so that there are no changes required to the tests.

You could definitely say I should have separated this work up differently.